specs/tabs/tabs.md

```diff
@@ -0,0 +1,173 @@
+- **Tab activation:** Offers a way to activate a tab and it's content by either focusing a tab or by key press (Space or Enter).
+- **Supplemental content:** Offers a way to add content to the left and/or right of the the tab list.
+
+### Risks and Challenges
+
+*Tabs* has specific guidance on DOM structure by the [WC3](https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html). This structure lacks logical groupings by separating the tab from its content. Some component libraries compensate for this by creating some kind of intermediary grouping that makes it easier for app authors to implement. Even though the DOM structure disassociates that logical grouping. While other component libraries stick to the DOM structure model. 
+
+### Prior Art/Examples
+- [FAST-DNA Pivot (React)](https://explore.fast.design/components/pivot)
+- [Material UI](https://material-ui.com/components/tabs/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/tabs/)
+- [Ant Design](https://ant.design/components/tabs/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/tabs)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tab-view)
+
+---
+
+### API
+
+*Component name:*
+- `fast-tabs`
+
+*Attributes:*
+- `orientation` - enum
+  - horizontal - default
+  - vertical
+- `activeId` - string
+- `tab-activation` - enum 
+  - focus - default
+  - key-press
+
+*Events:*
+- `onChange` - fires when component `activeId` updates
+
+### Anatomy and Appearance
+
+*Parts:*
+- tabs
+- tab-list
+- tab
+- tab-panel
+
+*Slot Names*
+- before-content
+- after-content
+
+*Template:*
+```HTML
+<div class="tabs" class="tabs">
+    <div class="tab-list" part="tab-list" role="tablist" aria-label="Sample Tabs">
+        <slot name="before-content" part="before-content"></slot>
+        <button class="tab" part="tab" role="tab" aria-selected="true" aria-controls="panel-1" id="tab-1" tabindex="0">
+            Tab One
+        </button>
+        <button class="tab" part="tab" role="tab" aria-selected="false" aria-controls="panel-2" id="tab-2" tabindex="-1">
+            Tab Two
+        </button>
+        <button class="tab" part="tab" role="tab" aria-selected="false" aria-controls="panel-3" id="tab-3" tabindex="-1">
+            Tab Three
+        </button>
+        <slot name="after-content" part="after-content"></slot>
+    </div>
+    <div class="tab-panel" part="tab-panel" id="panel-1" role="tabpanel" tabindex="0" aria-labelledby="tab-1">
+        Content of the first tab
+    </div>
+    <div class="tab-panel" part="tab-panel" id="panel-2" role="tabpanel" tabindex="0" aria-labelledby="tab-2" hidden>
+        Content of the second tab
+    </div>
+    <div class="tab-panel" part="tab-panel" id="panel-3" role="tabpanel" tabindex="0" aria-labelledby="tab-3" hidden>
+        Content of the third tab
+    </div>
+</div>
+```
+
+## Implementation
```

Comment:
Do you have a proposal for which implementation would work best?

Reply:
I personal like logical grouping option 2, as @EisenbergEffect 
noted above we could just have `<fast-tabs>` and `<fast-tab-item>` where the tab and tab content are slotted elements on `<fast-tab-item>` something like:

```
<fast-tabs>
    <fast-tab-item>
        <span slot="tab">Tab One</span>
        Content of the first tab
    </fast-tab-item>
    <fast-tab-item>
        <span  slot="tab">Tab Two</span>
        Content of the Second tab
    </fast-tab-item>
    <fast-tab-item>
        <span  slot="tab">Tab Three</span>
        Content of the Third tab
    </fast-tab-item>
</fast-tabs>
```

Reply:
Another option for the html could be to move the tab and panel out.

```html
<fast-tabs>
        <fast-tab>Tab One</fast-tab>
        <fast-tab-panel>Content of the first tab</fast-tab-panel>

        <fast-tab>Tab Two</fast-tab>
        <fast-tab-panel>Content of the Second tab</fast-tab-panel>

        <fast-tab>Tab Three</fast-tab>
        <fast-tab-panel>Content of the Third tab</fast-tab-panel>
</fast-tabs>
```

Reply:
With this configuration, we don't need to do any shuffling ourselfs. We can make the fast-tab go to the "tab" slot and fast-tab-panel go to the default slot.

Reply:
@chrisdholt @nicholasrice do you guys have thoughts here?

Reply:
Can we try this in the proof of concept suggested above? I think that'll give us the best lead on any potential drawbacks or limitations regarding implementation and API